### PR TITLE
I filled in my particulars and included my image already

### PR DESCRIPTION
**Pull request by dhs17y1jLimYuWei**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

**Comment by codedsun** [comment]
@dhs17y1jLimYuWei Please squash your commits!

**Review by nikit19 — CHANGES_REQUESTED** [review]
Looks good ! Just make sure that the dimensions of your image is 240 x 240 pixels

File: _data/students.yml [hunk 1]
```diff
@@ -297,3 +297,10 @@
   github: Akharote1
   image: aditya_kharote.jpg
   twitter: Akharote1
+
+- name: Lim Yu Wei
+  country: Singapore
+  github: dhs17y1jLimYuWei
+  image: limyuwei04.jpg
+  twitter: @MrNobodi123
```
Comment (nikit19) [inline _data/students.yml]: There is no @ in your username if I am right?
Please remove the @ from there

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
UPDATE your PR

**Review by ParthS007 — COMMENTED** [review]
@dhs17y1jLimYuWei Please resize your image to `240*240`  and update your PR 👍

**Review by anshumanv — COMMENTED** [review]
Upload image of 240x240 and resolve conflicts

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Pls resize your image to `240x240px`

**Comment by harshit98** [comment]
Good job! You're almost there to complete this pull request. Resize your image dimensions to 240 x 240.

**Review by isuruAb — COMMENTED** [review]
please resolve conflicts  and check the travis failure

**Review by ParthS007 — COMMENTED** [review]
@dhs17y1jLimYuWei  Closing this PR due to many conflicts and inactivity.Please ask on FOSSASIA Gitter channel if you face any difficulty 👍

**Pull request closed without merging**